dataplane/vrf.cpp

```diff
@@ -0,0 +1,399 @@
+#include <queue>
+
+#include "vrf.h"
+
+std::string ip4str(uint32_t ipAddress)
+{
+	return common::ipv4_address_t(ipAddress).toString();
+}
+
+std::string ip6str(const uint8_t* ipAddress)
+{
+	return common::ipv6_address_t(ipAddress).toString();
+}
+
+namespace dataplane::vrflpm
+{
+
+template<typename Allocator>
+void UpdateStats(Allocator& allocator, stats_t& stats)
+{
+	auto [blocks_total, blocks_used] = allocator.Stat();
+	stats.extended_chunks_size = blocks_total;
+	stats.extended_chunks_count = blocks_used;
+}
+
+/*
+ * Implementation VrfLpm4Linear
+ */
+
+VrfLpm4Linear::VrfLpm4Linear(BlocksAllocator& allocator) :
+        allocator_(allocator)
+{
+	Clear();
+}
+
+eResult VrfLpm4Linear::Insert(stats_t& stats, tVrfId vrfId, const uint32_t& ipAddress, const uint8_t& mask, const uint32_t& valueId)
+{
+	if (vrfId >= YANET_RIB_VRF_MAX_NUMBER)
```

Comment:
use `extended_chunks_size_min` instead of `YANET_RIB_VRF_MAX_NUMBER`, we done
`constexpr static uint64_t extended_chunks_size_min = YANET_RIB_VRF_MAX_NUMBER;` for a reason

And in 13 other places in this commit where `YANET_RIB_VRF_MAX_NUMBER` gets used